app/javascript/channels/client_channel.js

```diff
@@ -12,9 +13,8 @@ export const callback = {
   },
 
   received(data) {
-    let selector = data[0];
-    let newContent = data[1];
+    let newContent = data[0];
 
-    document.querySelector(selector).innerHTML += newContent;
+    document.querySelector("ul[data-js='messages-pub-sub']").innerHTML += newContent;
```

Comment:
Super minor, but it's occurring to me that if `getChannelName` were a method in the same object as `received`, we could also have a method for retrieving the DOM element or set the DOM element in a JS variable.